modules/client-configuration/pages/delete-clients.adoc

```diff
@@ -19,14 +28,187 @@ For more information about the system set manager, see xref:client-configuration
 
 [IMPORTANT]
 ====
-It is not possible to automatically clean up a a traditional client after deleting it. You have to take care of this
-yourself. Furthermore cleaning up a Salt minion does not remove Salt itself.
+It is not possible to automatically clean up a traditional client after deleting it.
+You have to take care of this yourself.
+Furthermore, cleaning up a Salt client does not remove Salt itself.
+====
+
+[NOTE]
+====
+Normally you migrate a traditional client to a Salt client without deleting the client.
+Salt automatically detects that you have a traditional client and does the necessary changes itself.
+But if you already deleted the traditional client and want to register it as a Salt client again, see
+xref:administration:troubleshooting/tshoot-register-trad-as-salt-after-deletion.adoc[].
+====
+
+
+
+== Delete a Salt Client on the Command Line (with API Call)
+
+.Procedure: Deleting a Client from the Server
+
+. Delete the client with the FQDN (Fully Qualified Domain Name):
++
+----
+spacecmd system_delete FQDN
+----
+
+. Remove the Salt key with the FQDN:
++
+----
+salt-key -d FQDN
+----
+
+Sometimes a new registration of a deleted (unregistered) client might not be possible.
+To solve this issue, some Salt cache files should be deleted on the {productname} Server (Salt master) before trying to re-register again:
+
+----
+rm /var/cache/salt/master/thin/version
+rm /var/cache/salt/master/thin/thin.tgz
+----
+
+
+
+[[delete.clients.commandline]]
+== Delete a Client from the Command Line
+
+
+=== Salt Client
+
+// Manual Registration Cleanup
+
+This process is only for {productname} clients, do not run it on the {productname} Server itself.
+
+[NOTE]
+====
+You must not execute the following procedure on clients running {rhel} and clones.
```

Comment:
With such a note I would ask "and how I do this on RHEL....?".
Any reason why we do not document RHEL or Debian ?
I think it is pretty similar and only zypper needs to be replaced by yum/dnf or apt.

Reply:
ok, I'll submit a proposal.